lib/common/ipc_client.c

```diff
@@ -1345,16 +1344,6 @@ crm_ipc_send(crm_ipc_t *client, const xmlNode *message,
         pcmk__clear_ipc_flags(flags, "client", crm_ipc_client_response);
     }
 
-    if(header->size_compressed) {
```

Comment:
This could go in the next commit where we're removing all the other checks, but doesn't matter